x-pack/plugins/infra/public/alerting/log_threshold/components/alert_dropdown.tsx

```diff
@@ -31,13 +30,14 @@ export const AlertDropdown = () => {
   const {
     services: {
       application: { capabilities },
+      observability,
     },
   } = useKibanaContextForPlugin();
```

Comment:
Same here ...although this existed before so I think it's OKish to leave it as it is. (I will eventually remove it)